common/flatpak-oci-registry.c

```diff
@@ -288,6 +291,17 @@ flatpak_oci_registry_set_token (FlatpakOciRegistry *self,
                                          0, NULL, NULL);
 }
 
+void
+flatpak_oci_registry_set_signature_lookaside (FlatpakOciRegistry *self,
+                                              const char         *signature_lookaside)
+{
+  g_set_str (&self->signature_lookaside, signature_lookaside);
+
+  if (self->signature_lookaside != NULL &&
+      g_str_has_suffix (self->signature_lookaside, "/"))
+    self->signature_lookaside[strlen (self->signature_lookaside) - 1] = '\0';
```

Comment:
This isn't wrong, but its hardly the most elegant way to remove a trailing /.

Reply:
Honestly struggling to find a nice way to do this. None of the glib functions seem to make it easier, so the best I could come up with is:
```c
  if (self->signature_lookaside != NULL)
    {
      size_t last = strlen (self->signature_lookaside) - 1;

      if (self->signature_lookaside[last] == '/')
        self->signature_lookaside[last] = '\0')
    }
```